src/highlighter/svg_highlighter.js

```diff
@@ -49,10 +49,23 @@ sre.SvgHighlighter.prototype.highlightNode = function(node) {
     node.style.color = this.colorString().foreground;
     return info;
   }
-  var bbox = node.getBBox();
-  var rect = document.createElementNS(
-      'http://www.w3.org/2000/svg', 'rect');
-  var padding = 40;
+  var rect = document.createElementNS('http://www.w3.org/2000/svg','rect');
+  var padding = 40, bbox;
+  if (node.nodeName === "use") {
+    //
+    //  WebKit has a bug where the x and y attributes for a <use> element
+    //  are not taken into account in the getBBox() call
+    //  (see https://code.google.com/p/chromium/issues/detail?id=512081)
+    //  so we temporarily wrap the <use> in a <g> and use getBBox() on that.
+    //
+    var g = document.createElementNS('http://www.w3.org/2000/svg','g');
+    node.parentNode.insertBefore(g,node);
+    g.appendChild(node);
+    bbox = g.getBBox();
+    g.parentNode.replaceChild(node,g);
+  } else {
+    bbox = node.getBBox();
+  }
   rect.setAttribute('x', bbox.x - padding);
```

Comment:
I actually did my original editing and seeing on the minified version (since I don't have Google closure installed, and didn't want to take time to do so and figure out how to use it), and the minified version used `'` rather than `"`, and did put more than one variable for a single `var` command.  

Note that there is `'` rather than `"` on lines 69, 70, and 71, so I thought that was accepted.